docs/https_source_interface.rst

```diff
@@ -50,39 +50,60 @@ you will be required both to confirm your affiliation with the organization,
 and to demonstrate control over the Onion URL for your Source Interface.
 
 In order for you to demonstrate control over the Onion URL for your Source
-Interface, DigiCert will provide you with some text and ask you to make it
-available at a `specific URL`_:
-``<onion_url>/.well-known/pki-validation/<unique_hash>.txt``.
-We have support for this workflow:
+Interface, you will need to perform a signing operation leveraging the
+private key of the Onion service used on the Source Interface.
+DigiCert will provide you with some text and request that you use that text
+in a signing operation. At a high level, obtaining a certificate from DigiCert
+involves:
+
+1. Generating an HTTPS keypair and CSR via ``openssl``.
+2. Submitting the CSR to DigiCert.
+3. Scheduling a phone call and verifying your relationship to the organization.
+4. Generating another CSR, using a custom tool, leveraging the Onion service private key.
+5. Submitting the second CSR to DigiCert.
+6. Downloading the certificate from the DigiCert panel.
+7. Installing the cert on the SecureDrop Application Server, via ``securedrop-admin``.
+
+For SecureDrop, you should perform these steps on the Admin Workstation.
+Below are detailed steps for use on Tails:
 
 .. code:: sh
 
-    # From the Admin Workstation, SSH to the Application Server
-    $ ssh app
+    # On the Admin Workstation, generate the first CSR
+    $ mkdir ~/Persistent/sd-https-key-generation
+    $ cd ~/Persistent/sd-https-key-generation
+    $ openssl req -new -newkey rsa:4096 -nodes -keyout sd.key -out sd.csr
```

Comment:
IMO it needs to be explicit that this is the key you'll be using with the final cert - anecdotally, because of the second CSR generated for domain validation purposes, this has been a point of confusion. So wording like "Save this key, as you will need it to use the certificate issued by Digicert" or something would be a good addition.